### PR TITLE
Phosphorus removal test

### PR DESCRIPTION
**Pull request by aladshaw3**
## Fixes/Addresses:

This PR establishes a preliminary test for a phosphorus removal unit process. 

## Summary/Motivation:

Phosphorus removal is one of the major pretreatment processes we need to simulate. This is a relatively complicated chemical problem involving speciation of iron and phosphorus with base buffers to raise pH and induce precipitation of FePO4. The model is extremely sensitive to tolerances associated with the safe log function in IDAES. Further refinement may be necessary to simulate a wider range of iron and phosphate concentrations. 

## Changes proposed in this PR:
- Adds a <code>test_phosphorus_removal.py</code> unit test

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: proteuslib/tests/test_phosphorus_removal.py [hunk 1]
```diff
@@ -0,0 +1,1205 @@
...
+                    "mw": (152.8, pyunits.g/pyunits.mol),
+                    "dens_mol_liq_comp_coeff": (55.2, pyunits.kmol*pyunits.m**-3),
+                    "cp_mol_liq_comp_coeff": (635000, pyunits.J/pyunits.kmol/pyunits.K),
+                    # NOTE: these parameters below are unknown
+                    "enth_mol_form_liq_comp_ref": (0, pyunits.kJ/pyunits.mol),
+                    "entr_mol_form_liq_comp_ref": (0, pyunits.J/pyunits.K/pyunits.mol)
+                                },
+                    # End parameter_data
+                    },
+        #NOTE: This test does NOT evaluate the precipitation function of IDAES
```
Comment (TimBartholomew) [inline proteuslib/tests/test_phosphorus_removal.py:568]: I'd move this note to the top of the file

**Review by TimBartholomew — APPROVED** [review]
Very minor comment, otherwise looks good to me

File: proteuslib/tests/test_phosphorus_removal.py (hunk 1 above)
Comment (aladshaw3) [reply proteuslib/tests/test_phosphorus_removal.py]: It has been moved.

**Review by TimBartholomew — APPROVED** [review]
LGTM

**Review by andrewlee94 — APPROVED** [review]
A quick look seems OK.

**Pull request merged**